### PR TITLE
main: Fix the test name as can be running in idea.

### PR DESCRIPTION
**Pull request by wayilau**
As original name occured error in idea,not correctly, so changed.

### CODE REVIEW

File: main_test.go [hunk 1]
```diff
@@ -22,7 +22,8 @@ import (
 	"testing"
 )
 
-func TestMain(t *testing.T) {
+
+func Test_Main(t *testing.T) {
```
Comment (spzala) [inline main_test.go:26]: @wayilau thank you for the PR but I didn't understand the motivation behind the changes. Please reopen with more information if needed. Thanks!

Comment (wayilau) [reply main_test.go:26]: Because i want run tests in IDE, but it can not be executed. so i changed the method name,  it works. @spzala

Comment (spzala) [reply main_test.go:26]: @wayilau hmmm...OK, I will keep it open for review, can you please add more description, any error that you are getting etc. Thanks!

Comment (wayilau) [reply main_test.go:26]: it tells me that it is a wrong test signature, so i can not run it in the ide. after i change it to Test_Main, it works. As TestMain is a test function used for testing.M, so we can use  the 'Run' Method to tests our  code,  the goal is  init or clean resourcds,  but  here we are not. so i changed it  to Test_Main. @spzala

**Comment by wayilau** [comment]
Anyone help me to check this pr ?

**Pull request closed without merging**